react/features/display-name/functions.js

```diff
@@ -7,7 +7,7 @@
  * @param {string} suffix - Suffix that will be appended.
  * @returns {string} The formatted display name.
  */
-export function appendSuffix(displayName: string, suffix: string) {
+export function appendSuffix(displayName: string, suffix: ?string) {
```

Comment:
wouldn't it be better to give it a default value of `''` ?

Reply:
Yes. That would reduce the perceived variable arity, which I feel would be simpler to read.